src/services/music/MusicUtils.ts

```
@@ -0,0 +1,26 @@
+  const regex = /(?:https?:\/\/)?(?:www\.)?(?:youtube\.com\/watch\?v=|\/watch\?v=|\/embed\/)([a-zA-Z0-9_-]{11})/;
+  const match = url.match(regex);
+  return match ? match[1] : null;
+}
+
+function areMembersInSameVoiceChannel(member: GuildMember, botMember: GuildMember): boolean {
+  const memberChannel = member.voice.channel;
+  const botChannel = botMember.voice.channel;
+  return memberChannel && botChannel && memberChannel.id === botChannel.id;
+}
+
+function findFirstVoiceChannel(client: Client, guildId: string) {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)